git-and-github/git.md

```diff
@@ -108,6 +108,9 @@ The above simple commands will be about 95% of what you will do with git. As lon
 
 This should give you a better idea of what it is exactly that you want to fix and should help you describe what you want to do. If you can describe your problem well then the internet (or ChatGPT) will always have an answer.
 
+You can also find concise guidelines on writing clear, human- and machine-readable commits on the [Conventional Commits site](https://www.conventionalcommits.org/en/v1.0.0/).
```

Comment:
What is the purpose of dash after the "human" keyword? I might not be seeing it properly as reviewing from my mobile phone.

Reply:
Thank you @allhandsondeck. Yes, you are right.  I meant "human-readable and machine-readable commits", but that might not have been clear enough.  Let me write it in the different way.

"You can also find concise guidelines on writing clear, human-readable and machine-readable commit messages on the [Conventional Commits site](https://www.conventionalcommits.org/en/v1.0.0/)."

**or** 

"You can also find concise guidelines on writing clear, well-structured commit messages on the [Conventional Commits site](https://www.conventionalcommits.org/en/v1.0.0/)."

**Which one do you find better?**

Reply:
I like
> "You can also find concise guidelines on writing clear, well-structured commit messages on the Conventional Commits site."

Reply:
@stasel I have added an additional commit with updates. Please review the changes.